Dockerfile

```diff
@@ -1,22 +1,15 @@
-# base build
-FROM node:8.9.4 AS base
+FROM node:8.9.4
+
```

Comment:
This change should resally be on separate branch or develop - it's confusing to have it here as not part of change. I know it was Adam triggered it!